Source/utils/sdl2_to_1_2_backports.cpp

```diff
@@ -779,6 +779,9 @@ char *SDL_GetBasePath()
 #if defined(__3DS__)
 	retval = SDL_strdup("file:sdmc:/3ds/devilutionx/");
 	return retval;
+#elif defined(__amigaos__)
+	retval = SDL_strdup("PROGDIR:");
+	return retval;	

```

Comment:
Trailing space (this is what is causing clang-format to fail)
```suggestion
	return retval;
```